src/main/scala-2.11/markets/exchanges/ExchangeLike.scala

```diff
@@ -25,9 +28,23 @@ trait ExchangeLike {
     */
   def settlementMechanism: ActorRef
 
-  def marketActorFactory(tradable: Tradable): ActorRef = {
-    val marketProps = MarketActor.props(matchingEngine, settlementMechanism, tradable)
-    context.actorOf(marketProps, tradable.ticker)
+  def marketActorFactory(ticker: Agent[Tick], tradable: Tradable): ActorRef = {
+    val marketProps = MarketActor.props(matchingEngine, settlementMechanism, ticker, tradable)
+    context.actorOf(marketProps, tradable.symbol)
+  }
+
+  def exchangeActorBehavior: Receive = {
+    case order: Order =>  // get (or create) a suitable market actor and forward the order...
+      val market = context.child(order.tradable.symbol).getOrElse {
+        val ticker = ???  // @todo create new symbol!
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Usage of ??? operator](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1560525571/issues/source?bid=2921361&fileBranchId=3019109#l39)